pkg/metricsprovider/k8s.go

```diff
@@ -36,8 +36,10 @@ var (
 )
 
 const (
-	localKubeConfig = "/Users/aqadeer/.kube/config" // path to local kube config file
-	localDev        = "LOCAL_DEV"                   // env variable to indicate client is outside the cluster usu. for local testing
+	// path to local kube config file
```

Comment:
It would be nice if it can take kube config as an optional argument instead of hard coding it as a const.

Reply:
Fixed it.